packages/website/docs/Button.md

```diff
@@ -1,39 +1,46 @@
-## Guidelines
+## Best Practices
 
-### General
+Button label should allow users to **foresee what will happen** when clicking them. 
 
--   **To redirect** users to another page of the current website or app, or to another website, **use links** rather than buttons.
--   Avoid multiplying buttons within a given page. If you need several buttons, less popular or **less important actions may be grouped** in a ["more" Menu button](../#/components/Menu) (dropdown list) or visually styled as isolated actions or links.
+Avoid multiplying buttons within a page. A page should have **only one primary action button**. If several secondary action buttons are required, consider using [Actionable items](#/form/ActionableItem) instead.
 
-### Labels
+When using button groups, such as for "Save" and "Cancel" actions, always put the action that move the user forward through his journey **to the right**. For instance, order buttons as "Cancel" then "Save".
 
--   The button label should allow users to **foresee what will happen** when clicking it.
--   Button labels should be **short** and use **trigger words**.
--   As much as possible, the **first word** of the label should be an **action verb** (e.g., the label for a button that allows the user to add a filter could be "Add Filter").
 
-### Alterative Texts
+## Labeling
 
--   When a button includes **only an icon** (i.e., no label), **label information** should be **added in the alternative text** element in the HTML and appear **in a tooltip** when hovering the button.
--   When a button contains a **text as well as an icon**, the icon is a visual help to identify the purpose faster. Providing an **alternative text is not recommended** as it would only repeat the information when using a **screenreader**.
+Keep labels short, preferably **under three words**.
+Use **trigger words** to clearly indicate the action performed by the button.
 
-## Variations
+Write labels so **the first word is a verb**. Use noun without article, to specify the context when required. For example, write "Add filter" rather than "Add a filter" but only write "Save" and "Cancel".
 
-The button type choice depends on the function of the button (e.g., primary action, secondary action), the number of actions available in the screen (e.g., one, two, several), and its location (e.g., header, action bar, page, footer).
+When using button to confirm an action that will be executed, always use an active word to **clearly state the resulting action**. For instance, if user must confirm the deletion of a file, write "Delete" and "Cancel" rather than "Yes" and "No".
```

Comment:
```suggestion
When using a button to have users confirm the action to execute, use an active word to **clearly state the resulting action**. For instance, if users must confirm the deletion of a file, write "Delete" and "Cancel" rather than "Yes" and "No".
```